src/app/examples/examples.state.ts

```diff
@@ -1,5 +1,7 @@
 import { createFeatureSelector, ActionReducerMap } from '@ngrx/store';
 
+import { AppState } from '@app/core/core.state';
```

Comment:
In `CoreModule` we're still using `index.ts` file which re-exports everything so we can simplify this import as `'@app/core'`. Maybe we will move away from `index.ts` barrels in the future. They were promoted in the beginning, then not and now I think they are still promoted for `library` style packages with `public_api.ts` or something similar.

Reply:
I think it's also used with the settings and shared module.
I think with the `paths` defined in `tsconfig.json`, we don't need these `index` files anymore?

Reply:
In settings we have for example in `settings.effects.ts` import like this `import { LocalStorageService, AnimationsService } from '@app/core';` 

I think that for core it's also pretty nice, we're not interested in the internal structure of core so we're fee to shuffle things around inside as long as the exported services still exists. 

Other modules / services usually import only their own stuff which is always imported relatively and besides that they consume `CoreModule` so in that case `index.ts` and short import paths can make sense hmm?

Reply:
Also now I realized, I forgot to refactor settings to split out selectors / actions / etc... But will do it after this is finished :)

Reply:
After reading your comment it makes sense to have an `index` for `CoreModule` 👍